target/linux/ath79/image/generic.mk

```diff
@@ -244,6 +244,15 @@ define Device/comfast_cf-e5
 endef
 TARGET_DEVICES += comfast_cf-e5
 
+define Device/compex_wpj531
+  ATH_SOC := qca9531
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
+  IMAGE_SIZE := 16128k
+  DEVICE_TITLE := Compex WPJ531 (16MB flash)
```

Comment:
Please use the new syntax:
DEVICE_VENDOR := Compex
DEVICE_MODEL := WPJ531
DEVICE_VARIANT := 16M

Reply:
```
diff --git a/target/linux/ath79/image/generic.mk b/target/linux/ath79/image/generic.mk
index 6df0ea7e48..9d459a8429 100644
--- a/target/linux/ath79/image/generic.mk
+++ b/target/linux/ath79/image/generic.mk
@@ -261,8 +261,9 @@ define Device/compex_wpj531
   ATH_SOC := qca9531
   DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
   IMAGE_SIZE := 16128k
-  DEVICE_TITLE := Compex WPJ531 (16MB flash)
-  BOARDNAME := WPJ531
+  DEVICE_VENDOR := Compex
+  DEVICE_MODEL := WPJ531
+  DEVICE_VARIANT := 16M
   IMAGES += cpximg-7A03.img cpximg-7A04.img cpximg-7A06.img cpximg-7A07.img
   IMAGE/cpximg-7A03.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x68a 2
   IMAGE/cpximg-7A04.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x693 3
```